diesel/src/result.rs

```diff
@@ -82,7 +82,26 @@ pub enum ConnectionError {
 pub type QueryResult<T> = Result<T, Error>;
 pub type ConnectionResult<T> = Result<T, ConnectionError>;
 
+/// See the [method documentation](#tymethod.optional).
```

Comment:
This link depends on the way rustdoc adds ids to certain elements on the page. I don't think this is public API and will work in future version of rustdoc. But let's keep it for now and hopefully just upgrade to glorious intra-rustdoc links later!